features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,18 @@
+require 'byebug'
+
+When('I fill in Correo with {string} and Contrasenia with {string} and click on {string} button') do |email, password, button|
```

Comment:
We usually prefer leaving these statements the most simple we can. Try something like "I fill the required fields". This way, it's more readable, and also you are not exposing sensitive information (you can store those email and password fields in dotenv).